src/packagedcode/conda.py

```diff
@@ -14,6 +14,7 @@
 
 from packagedcode import models
 from packagedcode.pypi import BaseDependencyFileHandler
+from dparse2.parser import parse_requirement_line
```

Comment:
Do you really want to use this and not our own pip-requirements-parser?

Reply:
This is what we were using previously with dparse, I added some improvements on the function, and this was using  `parse_requirement_line` internally, but `pip-requirements-parser` would make more sense possibly, I will open a follow up issue on conda to include this. This would be helpful also to parse requirement lines with source repos.